src/binaries/query/main.rs

```diff
@@ -237,12 +237,14 @@ async fn main_entrypoint() -> Result<()> {
             "unlimited".to_string()
         }
     });
+    println!("    global allocator: {}", GlobalAllocator::name());
```

Comment:
How about:

```shell
    allocator: jemalloc
        config: xxxxxxxxxxx
```

or 

```shell
    allocator: jemalloc (xxxxxxxxxxxx)
```

For better reading?

Reply:
```
Memory:
    limit: unlimited
    allocator: jemalloc
    config: percpu_arena:percpu,oversize_threshold:0,background_thread:true,metadata_thp:auto,dirty_decay_ms:5000,muzzy_decay_ms:5000
```